cmd/kubelet/app/server.go

```diff
@@ -175,6 +181,12 @@ func (s *KubeletServer) Run(_ []string) error {
 		HighThresholdPercent: s.ImageGCHighThresholdPercent,
 		LowThresholdPercent:  s.ImageGCLowThresholdPercent,
 	}
+
+	cloud := cloudprovider.InitCloudProvider(s.CloudProvider, s.CloudConfigFile)
+	if cloud != nil {
+		glog.Infof("Successfully initialized cloud provider: %s from the config file: %s\n", s.CloudProvider, s.CloudConfigFile)
```

Comment:
nit: I tend to use %q instead of %s since it makes empty and white-space values clear